django/db/models/base.py

```diff
@@ -660,6 +660,25 @@ def _set_pk_val(self, value):
                 setattr(self, parent_link.target_field.attname, value)
         return setattr(self, self._meta.pk.attname, value)
 
+    def _validate_force_insert(self, force_insert):
+        if isinstance(force_insert, bool) or not force_insert:
+            return force_insert
+
+        if not isinstance(force_insert, tuple):
+            raise ValidationError("force_insert must be a bool or tuple")
+
+        if any(not issubclass(item, Model) for item in force_insert):
```

Comment:
I think that the proper validation here is 

```suggestion
        if not all(isinstance(item, ModelBase) and issubclass(cls, item) for item in force_insert):
```

To ensure that `force_insert` doesn't include unrelated bases.